src/vmm/src/devices/virtio/mem/persist.rs

```diff
@@ -28,7 +28,7 @@ pub struct VirtioMemState {
     usable_region_size: u64,
     requested_size: u64,
     slot_size: usize,
-    plugged_blocks: BitVec,
+    plugged_blocks: Vec<bool>,
```

Comment:
Isn't there a way to ignore invalid `BitVec` objects generated from the fuzzer, to avoid the crashes? As you say it's not a big deal, but it's very annoying that we need to change a more efficient data structure just because the fuzzer doesn't know how to handle it.